README.md

```diff
@@ -150,7 +150,14 @@ The `OR` operator works like this:
 users.where(users[:name].eq('bob').or(users[:age].lt(25)))
 ```
 
-The `AND` operator behaves similarly.
+The `AND` operator behaves similarly. Here is an example of the `DISTINCT` operator, which as of v6 is also chainable:
```

Comment:
`Arel` itself... my source is [this comment](http://stackoverflow.com/questions/3136278/how-to-fetch-distinct-values-with-arel-relational-algebra/12485350#comment49130014_12485350) so I'm ok with removing that